test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -216,3 +200,56 @@ def hook(context):
 
     previousFailResultCount = test.resultCount("fails")
     previousErrorResultCount = test.resultCount("errors")
+
+
+def teardown_client():
+    # Cleanup user accounts from UI for Windows platform
+    # It is not needed for Linux so skipping it in order to save CI time
+    if not isWindows():
```

Comment:
?

Reply:
Right now, I am just trying to get the intel on by how much this will increase the CI time